.github/workflows/build_and_deploy.yml

```
@@ -214,6 +220,54 @@ jobs:
           files: |-
             ${{ steps.build-voicevox-core-python-api.outputs.whl }}
           target_commitish: ${{ github.sha }}
+  build_xcframework:
+    needs: build_and_deploy
+    runs-on: macos-12
+    steps:
+      - name: Set ASSET_NAME env var
+        shell: bash
+        run: echo "ASSET_NAME=voicevox_core-ios-xcframework-${{ env.VERSION }}" >> "$GITHUB_ENV"
```

あ、過去のコアの命名規則に合わせて`-cpu`もつけてあげても良いでしょうか 🙇‍♂️ 
```suggestion
        run: echo "ASSET_NAME=voicevox_core-ios-xcframework-${{ env.VERSION }}" >> "$GITHUB_ENV"
```

（例えばAndroidのは`voicevox_core-android-x86_64-cpu-[version].zip`という名前です）
https://github.com/VOICEVOX/voicevox_core/releases/tag/0.15.0-preview.3

たぶん大丈夫だと思うので勝手ながらsuggestをcommitしてマージします！